opbeans-dotnet/Startup.cs

```diff
@@ -37,7 +37,7 @@ public void ConfigureServices(IServiceCollection services)
 		// This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
 		public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 		{
-			app.UseElasticApm(Configuration);
+			app.UseAllElasticApm(Configuration);
```

Comment:
Just for future reference: as long as the new `Elastic.Apm.NetCoreAll` is not published to nuget.org, people won't be able to build the solution just by cloning the repo and clicking "build" in visual studio or by `dotnet build`.

In CI we build the source of the agent and create the `Elastic.Apm.NetCoreAll` package and pull it from the local nuget repo, so it'll work there.

Talked to @v1v  - we are good with this, this is a temporary situation and we accept it.